english/rib-eye/rib-eye.md

```diff
@@ -14,4 +14,8 @@ or
 
 or
 
+[In Camelot] (https://github.com/itsmeava/create-your-own-adventure/blob/master/english/camelot/camelot.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.